ocs_ci/ocs/registry.py

```diff
@@ -37,6 +37,15 @@ def change_registry_backend_to_ocs():
         resource_name=constants.IMAGE_REGISTRY_RESOURCE_NAME, params=param_cmd, format_type='json'
     ), f"Registry pod storage backend to OCS is not success"
 
+    if(config.ENV_DATA['platform'] not in constants.CLOUD_PLATFORMS):
+        run_cmd(
```

Comment:
Is the OK for OCP 4.2 on VMware, the docs in Fixes is pointing to OCP 4.3?

Reply:
I do not think that it will have any impact on 4.2 as this is suppose to be default value and in 4.2 it's working with such value by default:
```yaml
spec:
  defaultRoute: true
  disableRedirect: false
  httpSecret: 868041c963e68caa8eb5b8e7564fffe1c33400336c2db02dc8d4e8a006716b17c6ea74655a1da7ac066e7bfaf6a745f2a47c16491c1ae5b7cbee12869c04c40c
  logging: 2
  managementState: Managed
```